src/lsdb/catalog/catalog.py

```diff
@@ -123,6 +123,18 @@ def assign(self, **kwargs) -> Catalog:
 
         Returns:
             The catalog containing both the old columns and the newly created columns
+
+        Examples:
+            Create a new column using a function::
+
+                catalog = Catalog(...)
+                catalog = catalog.assign(new_col=lambda df: df['existing_col'] * 2)
+
+            Add a column from a 1-D Dask array:
```

Comment:
```suggestion
            Add a column from a 1-D Dask array::
```

Needed here as well.

Reply:
Ah, the perils of just viewing the suggestion snippet from the main page of the PR. Thanks.